### ISSUE TITLE
Adding AdPerformanceReportColumn.AdDescription2 field in AdPerformanceReportRequest throw null reference exception on GetStringValue 

### ISSUE THREAD

**Issue by momolasd** [BingAds/BingAds-dotNet-SDK#93]
Hi guys, I observe the following behavior when submitting an AdPerformanceReportRequest and I do not think it is the normal behavior or there is something that I do not understand in the report logic

this is my code

```
var adPerformanceReportRequest = new AdPerformanceReportRequest
{
    Aggregation = ReportAggregation.Summary,
    ExcludeColumnHeaders = false,
    ExcludeReportHeader = true,
    ExcludeReportFooter = true,
    Format = ReportFormat.Csv,
    ReturnOnlyCompleteData = true,
    Time = new ReportTime
    {
        PredefinedTime = (Microsoft.BingAds.V13.Reporting.ReportTimePeriod)reportTimePeriod,
        ReportTimeZone = ReportTimeZone.BrusselsCopenhagenMadridParis
    },
    Scope = new AccountThroughAdGroupReportScope
    {
        AccountIds = accountIds.ToArray(),
        Campaigns = null,
        AdGroups = null
    },
    Filter = new AdPerformanceReportFilter { },
    Columns = new[]
    {
        AdPerformanceReportColumn.AccountName,
        AdPerformanceReportColumn.AccountId,
        AdPerformanceReportColumn.AdId,
        AdPerformanceReportColumn.AdTitle,
        AdPerformanceReportColumn.Impressions,
        AdPerformanceReportColumn.AdDescription,
        AdPerformanceReportColumn.AdDescription2,
        AdPerformanceReportColumn.Clicks,
        AdPerformanceReportColumn.Spend,
        AdPerformanceReportColumn.FinalUrl,
        AdPerformanceReportColumn.AdGroupId,
        AdPerformanceReportColumn.AdGroupName,
        AdPerformanceReportColumn.AdGroupStatus,
        AdPerformanceReportColumn.Assists,
        AdPerformanceReportColumn.CampaignName,
        AdPerformanceReportColumn.Language,
    }
};

var reportDownloadParameter = new ReportingDownloadParameters
{
    ReportRequest = adPerformanceReportRequest,
    OverwriteResultFile = true
};

IEnumerable<IReportRecord> records = DownloadReportAsync(reportDownloadParameter).GetAwaiter().GetResult();

foreach (var record in records)
{
    string u = record.GetStringValue("AccountName");
    string s = record.GetStringValue(AdPerformanceReportColumn.AccountId.GetName());
}
```

So this is throwing an Null Reference Exception on the `string u = record.GetStringValue("AccountName");` line

After some research i found that if i remove the **AdPerformanceReportColumn.AdDescription2** field, everything works fine

Can u explain me if i'ts the normal behaviour what i missed or investigate if it's real issue ? 
Let me know if you need more repro details

Edit : I have the same behaviour with the TitlePart1, TitlePart2 and TitlePart3 fields

**Comment by qitia** [comment]
could you please share the trackingid/requestid of your DownloadReport Request

**Comment by bdetry** [comment]
Thank you @momolasd for opening this issue, **we actually face the same problem here.** 

 The request works fine if we request the following "columns" 

`            var columns = new[]
            {
                AdPerformanceReportColumn.AccountName,
                AdPerformanceReportColumn.AccountId,
                AdPerformanceReportColumn.AccountNumber,
                AdPerformanceReportColumn.AdId,
                AdPerformanceReportColumn.AdGroupId,
                AdPerformanceReportColumn.AdTitle,
                AdPerformanceReportColumn.AdDescription,
                AdPerformanceReportColumn.AdType,
                AdPerformanceReportColumn.FinalUrl,
                AdPerformanceReportColumn.CampaignStatus,
                AdPerformanceReportColumn.Spend
            };`

 If we add any `AdDescription2` , `TitlePart1` (2 or 3) the request wont fail when we download the report using `ReportingServiceManager.DownloadReportAsync`

But fails when we execute the methode `.GetLongValue("AdId")` one of the records `DownloadReportAsync` gave us. (Type of the records `IReportRecord)`

**Really stange behavior :**  In my debug session I see my report record (`IReportRecord`)  has a private member called `rowValues` with another property `Columns`. 
**Depending on the columns I ask the first row returned by `DownloadReportAsync` will contain column names or values in `rowValues.Columns`. No matter what I set in `AdPerformanceReportRequest.ExcludeColumnHeaders` (the reporting options)**

Thank you @qitia

**Comment by bdetry** [comment]
Hi?

**Comment by qitia** [comment]
hey @bdetry ,  I will try to repro from my side, but if you can provide trackingid/requestid, that will be great helpful.

**Comment by qitia** [comment]
I confirm there is a bug here in [ReportHeader](https://github.com/BingAds/BingAds-dotNet-SDK/blob/83959d1397553c02cb9896995906db84f5ba180d/BingAdsApiSDK/V13/Internal/Reporting/ReportHeader.cs#L62), the regex should be 
```
private readonly Regex columnRegex = new Regex(@"^(([1-9]\d*.?\d*)|(0\.\d*[1-9]))%?$");
```

we will send a fix soon and will update when it is ready.

**Comment by bdetry** [comment]
Thank you very much @qitia

**Comment by bdetry** [comment]
Hello @qitia any update on the issue?

**Comment by qitia** [comment]
hi thanks for ping. We are going to publish new version before end of this week.

**Comment by qitia** [comment]
fixed with Version 13.0.11 which was published 08/20
